src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/CaseWorkerRefController.java

```diff
@@ -256,7 +256,7 @@
             consumes = APPLICATION_JSON_VALUE,
             produces = APPLICATION_JSON_VALUE
     )
-    @Secured("cwd-admin")
+    @Secured({"cwd-system-user"})
     public ResponseEntity<Object> fetchCaseworkersById(@RequestBody UserRequest userRequest) {
```

Comment:
required 2 roles ?
'cwd-read-only' role for the IAC service
'cwd-system-user' role for the ORM service (ORM service name am_org_role_mapping_service)

Reply:
For now, we need the system user role only.